uber/templates/preregistration/index.html

```diff
@@ -57,6 +57,11 @@
         <td>
             <ul style="padding-left:15px">
                 <li>Membership for {{ c.EVENT_NAME }}</li>
```

Comment:
We presumably want to explain as part of the checkout process why we're charging them less.